Modules/CodeTrek/Entities/CodeTrekCandidateFeedback.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Modules\CodeTrek\Entities;
+
+use Illuminate\Database\Eloquent\Model;
+
+class CodeTrekCandidateFeedback extends Model
+{
+    protected $fillable = ['candidate_id', 'posted_by', 'category_id', 'latest_round_name', 'feedback', 'feedback_type', 'posted_on'];
```

Comment:
If we are filling all the column then we can use `guarded` attribute
```suggestion
    protected $guarded = [];
```